### PR TITLE
Feature/boas 626 file size and type in document table

### PR DESCRIPTION
**Pull request by lutamKC**
## Describe your changes

change the Document table to store 2 new fields:
fileType - mime type string eg  application/pdf
fileSize - file size in bytes.

update associated apis to store / return this data.
Front end updated to pass this data to the api on upload.

## BOAS-626 Display File Type and File Size of docs in folder view page
https://pins-ds.atlassian.net/browse/BOAS-626
(raised against BOAS-584)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

[BOAS-626]: https://pins-ds.atlassian.net/browse/BOAS-626?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[BOAS-626]: https://pins-ds.atlassian.net/browse/BOAS-626?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[BOAS-584]: https://pins-ds.atlassian.net/browse/BOAS-584?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by francescobrenci-AR** [comment]
WEB looks good to me

File: apps/api/src/server/applications/application/documents/document.validators.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ const validateAllDocumentsExist = async (documentGuids) => {
 export const validateDocumentsToUploadProvided = composeMiddleware(
 	body('[]').notEmpty().withMessage('Must provide documents to upload'),
 	body('*.documentName').exists().withMessage('Must provide a document name'),
+	body('*.documentType').exists().withMessage('Must provide a document type'),
+	body('*.documentSize').exists().withMessage('Must provide a document size'),
```
Comment (mayya-bond-kc) [inline apps/api/src/server/applications/application/documents/document.validators.js:35]: (suggestion) I think if you wanted to, you could validate that this is a number (not a string for instance)

**Review by mayya-bond-kc — DISMISSED** [review]
Looking good! Teeny suggestion, not at all needs implementing, just jumped out at me.

**Review by francescobrenci-AR — APPROVED** [review]

File: apps/api/src/server/applications/application/documents/document.validators.js (hunk 1 above)
Comment (DavidJWhiteDev) [reply apps/api/src/server/applications/application/documents/document.validators.js:35]: good call, I will try and remember for next time

**Pull request merged**